### DISCUSSION TITLE
Update file in /public on production

### DISCUSSION THREAD

**Discussion by jonyw4** [vercel/next.js#20050] [Help] [answered]
Hi guys, 
Its possible to update an `.json` file that lives in Vercel? If yes how I can do that?

I have an `.json` file  on `/public` folder that I need to update  when in `POST` `/api/currency`  is requested. On localhost works, but in productions didn't work and returns: `Task timed out after 10.01 seconds`.

**Comment by jamesmosier** [comment, marked as answer]
Hi @jonyw4. The `public` folder is only meant to hold static assets prior to building, not during runtime. This [note](https://nextjs.org/docs/basic-features/static-file-serving) in the docs speaks to this a bit:

> Only assets that are in the public directory at build time will be served by Next.js. Files added at runtime won't be available. We recommend using a third party service like AWS S3 for persistent file storage.
